centos_ci.sh

```diff
@@ -3,6 +3,7 @@
 # bundle location
 BUNDLE_VERSION=4.2.7
 BUNDLE=crc_libvirt_$BUNDLE_VERSION.crcbundle
+GO_VERSION=1.12.8
```

Comment:
If we pick an upstream release rather than something curated by a distro, this really should be the latest one, so 1.12.13 in order to benefit from any important bug fix/CVE fix which was done since 1.12.8.